src/Components/forms/inputs/AutoFillDropDown.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import { Label } from 'reactstrap'
+
+export default ({ employerOnChange, isEmpty, label, name, arrayList }) => {
```

Comment:
Components should generally have _names_ (it looks like a lot of the others don't either, but it means your React Dev Tools is full of "Anonymous" entries). Either:
```suggestion
export default function AutoFillDropDown({ employerOnChange, isEmpty, label, name, arrayList }) {
```
or:
```suggestion
const AutoFillDropDown = ({ employerOnChange, isEmpty, label, name, arrayList }) => {
```
then `export default AutoFillDropDown`.

Reply:
applied the required changes